peru/main.py

```diff
@@ -221,6 +221,31 @@ def do_override(params):
             print('{}: {}'.format(module, params.runtime.get_override(module)))
 
 
+@peru_command('module', '''\
+Usage:
+    peru module <list> [-z]
+
+Options:
+    -z              print one module per line for machine-readable formatting
+
+''')
+def do_list(params):
+    modules = params.scope.modules.values()
+    output = ''
+    delim = ' | ' if params.args['-z'] else "\n  "
```

Comment:
I think the convention with `find` and `git` is to use the null character `\x00` as a delimiter in the machine-readable mode. That makes it at least a little less likely that one of the strings you're printing will accidentally contain the delimiter inside of it. (For example, if we were listing full targets like `module_name|rule_name`, the `|` in there would kind of conflict with the ` | ` delimiter we have here.)